tests/routes/test_route_objects.py

```diff
@@ -0,0 +1,89 @@
+import geopandas as gpd
+import pytest
+
+import routes.route as route
+
+
+def test_create_route_object(monkeypatch: pytest.MonkeyPatch) -> None:
+    def _get_num_people_on_route(
+        route_path: list[str], population_data: gpd.GeoDataFrame, cars_per_person: float
+    ) -> int:
+        return 10
+
+    def _get_total_population(
+        population_data: gpd.GeoDataFrame, cars_per_person: float
+    ) -> int:
+        return 30
+
+    monkeypatch.setattr(route, "_get_num_people_on_route", _get_num_people_on_route)
+    monkeypatch.setattr(route, "_get_total_population", _get_total_population)
+    list_of_paths = [["1", "2"], ["1", "2"], ["1", "2"]]
+    population_data = gpd.GeoDataFrame()
+    start = 0
+    end = 1000
+    cars_per_person = 1.0
+    create_route_objects = route.create_route_objects(
+        list_of_paths, population_data, start, end, cars_per_person
+    )
+    assert len(create_route_objects) == 3
+    assert create_route_objects[0].num_people_on_route == 10
+    assert create_route_objects[1].num_people_on_route == 10
+    assert create_route_objects[2].num_people_on_route == 10
+
+
+def test_create_route_object_uneven(monkeypatch: pytest.MonkeyPatch) -> None:
```

Comment:
Just so I'm not missing anything, are routes with an even or odd number of people on them special cases, or is this just an extra test?